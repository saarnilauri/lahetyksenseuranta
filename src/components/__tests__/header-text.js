import React from "react";
import renderer from "react-test-renderer";

import HeaderText from "../header-text.js";

describe("HeaderText", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<HeaderText />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
