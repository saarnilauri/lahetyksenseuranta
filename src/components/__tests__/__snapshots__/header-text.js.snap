// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderText renders correctly 1`] = `
<div
  className="header-text pb-5 md:pb-20 lg:pb-28 pt-20 mb-5 md:mb-20 px-5"
>
  <div
    className="grid grid-flow-col auto-cols-max mt-5 md:mt-24 lg:mt-15 xl:mt-52 mb-4"
  >
    <div
      className="lg:pl-7"
    >
      <svg
        className="w-5 md:w-10 h-auto lg:w-20"
        height="100"
        viewBox="0 0 60 100"
        width="60"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="#fff"
        >
          <path
            d="m59.543 46.694c0-1.618-1.312-2.93-2.93-2.93s-2.93 1.312-2.93 2.93c0 13.185-10.726 23.912-23.912 23.912-13.185 0-23.912-10.727-23.912-23.912 0-1.618-1.311-2.93-2.929-2.93s-2.93 1.312-2.93 2.93c0 15.427 11.796 28.15 26.842 29.627v6.052h-2.954c-8.105 0-14.698 6.593-14.698 14.697 0 1.618 1.312 2.93 2.93 2.93h35.303c1.618 0 2.929-1.312 2.929-2.93 0-8.104-6.593-14.697-14.697-14.697h-2.954v-6.052c15.046-1.477 26.842-14.2 26.842-29.627zm-15.549 47.447h-28.445c1.211-3.438 4.492-5.908 8.339-5.908h11.767c3.847 0 7.128 2.47 8.339 5.908z"
          />
          <path
            d="m29.771 64.697c9.727 0 17.64-7.913 17.64-17.639v-29.419c0-9.726-7.913-17.639-17.64-17.639-9.726 0-17.639 7.913-17.639 17.639v29.419c0 9.726 7.913 17.639 17.639 17.639zm0-58.838c6.496 0 11.78 5.285 11.78 11.78v.012h-3.16c-1.618 0-2.93 1.312-2.93 2.93s1.312 2.93 2.93 2.93h3.16v5.908h-3.16c-1.618 0-2.93 1.312-2.93 2.93s1.312 2.929 2.93 2.929h3.16v5.909h-3.16c-1.618 0-2.93 1.311-2.93 2.929s1.312 2.93 2.93 2.93h3.16v.012c0 6.496-5.284 11.78-11.78 11.78-6.495 0-11.78-5.284-11.78-11.78v-.012h3.161c1.618 0 2.929-1.312 2.929-2.93s-1.311-2.929-2.929-2.929h-3.161v-5.909h3.161c1.618 0 2.929-1.311 2.929-2.929s-1.311-2.93-2.929-2.93h-3.161v-5.908h3.161c1.618 0 2.929-1.312 2.929-2.93s-1.311-2.93-2.929-2.93h-3.161v-.012c0-6.495 5.285-11.78 11.78-11.78z"
          />
        </g>
      </svg>
    </div>
    <div
      className="ml-3 lg:ml-8"
    >
      <h1
        className="text-white font-bold text-xl md:text-5xl md:mt-2 lg:text-6xl lg:mt-6 xl:text-7xl tracking-tight relative text-shadow-md"
      >
        Lähetyksenseuranta podcast
      </h1>
    </div>
  </div>
  <p
    className="text-white text-lg md:text-3xl lg:text-4xl lg:ml-8 xl:text-4xl py-2 lg:py-5 px-7 text-shadow-md"
  >
    Podcast‚ joka käsittelee lähetystyötä nykyajassa.
  </p>
</div>
`;
